navapbc/wic-participant-recertification-portal
# PRP-351 Update database module to use RDS instead of Aurora

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-351

## Changes
* Added a new database child module `database-rds`
* moved all database associations to the new module

## Context for reviewers
> RDS Aurora serverless v2 is so expensive and it’s really not necessary for a project of this size. If we want other states to be able to use our repo, we could really help them out by providing a drop-in replacement for our current infra/modules/database module that uses a much cheaper offering in RDS.

## Testing
1. Log into the AWS console and find the `dev` database in the `RDS` section of the database console
2. Visit the dev portal and ensure that all activity is the same
